common/changes/@microsoft/rush/octogonz-heft-issue-4467_2024-10-01-08-08.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "",
+      "type": "none"
```

Comment:
rush changes are solely contract changes from `operation-graph`, would it suffice to use none here?

Reply:
Changes to Rush are always `none`; we configure it when we manually publish Rush.